man/systemd.unit.xml

```diff
@@ -1658,6 +1658,30 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ConditionMemoryPressure=</varname></term>
+          <term><varname>ConditionCPUPressure=</varname></term>
+
+          <listitem><para>Verify that the overall system (memory or CPU) pressure is below the given threshold.
+          This setting takes a colon-separated tuple as argument. The first value is the slice unit under which
+          the pressure will be checked, and it is optional: if not specified, the overall system pressure will be
+          measured, instead of a particular cgroup's.</para>
+
+          <para>The second value is the threshold. It can be specified as a simple percentage value, suffixed with
+          <literal>%</literal>, in which case the pressure will be measured as an average over the last five
+          minutes before the attempt to start the unit is performed, and it will be checked if it is strictly
+          below the specified threshold.</para>
+
+          <para>Alternatively, the average type and the comparison operator can also be specified, for example:
```

Comment:
alternative to what precisely? the first *and* second arg? or just the latter? please be specific here.